pkgs/misc/vscode-extensions/wakatime/default.nix

```diff
@@ -8,23 +8,15 @@ in
     mktplcRef = {
       name = "vscode-wakatime";
       publisher = "WakaTime";
-      version = "4.0.9";
-      sha256 = "0sm2fr9zbk1759r52dpnz9r7xbvxladlpinlf2i0hyaa06bhp3b1";
+      version = "17.0.8";
+      sha256 = "0bwnwnmkiv298d1fbd6i20c2k1s8lymhzmawva4zn6dckimllkd8";
     };
 
-    postPatch = ''
-      mkdir wakatime-cli
-      ln -s ${wakatime}/bin/wakatime ./wakatime-cli/wakatime-cli
```

Comment:
`wakatime` no longer a dependency in new version but still taken as a parameter.

Reply:
I would assume that the extension won't be functional as long as the `wakeatime` is not in `PATH`, right? I might be a good idea to provide nixpkgs's wakeatime as the default `wakeatime` exe if possible when none is explicitly specified via settings.